### ISSUE TITLE
Convergency of the model for VGG's synthetic dataset.

### ISSUE THREAD

**Issue by HotaekHan** [bgshih/crnn#132]
Hi, Thanks for your work :)

I'm trying to train CRNN for Synthetic data from VGG lab(about 7 milion) and Focused Scene Text data from ICDAR(about 800).
However, my train loss seems to be not converged. my loss is stuck in about 18.xxx ~ 20.xxx.
Can you share your hyperparameter or any tips?
And can you explain why your implementation is differenct with your paper? is it better performace?

Thanks :)
Best,

**Comment by haneSier** [comment]
Just set a learning rate that converged fastly in the beginning When I trained my Chinese characters recognition model , if encountering the loss was in plain，i would like to save the current model and reset a small learning rate then continued to train and attained an accuracy of 97.7% in a 3.6million Chinese characters data set（codes were open in my github repository ）
